src/widgets/histogram/histogram-title-view.js

```diff
@@ -38,7 +40,10 @@ module.exports = cdb.core.View.extend({
 
   _initBinds: function () {
     this.widgetModel.bind('change:title change:collapsed', this.render, this);
-    this.dataviewModel.bind('change:histogram_sizes', this.render, this);
+    this.dataviewModel.bind('change:autoStyle', function () {
+      this.render();
+      this.dataviewModel._onStyleChanged();
```

Comment:
This should be handled by CartoDB.js internally IMO. If `autoStyle` has changed, it means that the `cartoCSS` has been changed, right? Can we "detect" this in CartoDB.js at the layer level and reload the map if needed?